blog/blog.html

```diff
@@ -1,52 +1,22 @@
 <!DOCTYPE html>
-
-<html class="no-js" lang="en">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, maximum-scale=1"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+
+    <title>KyootyBellas blog</title>
 
-    <script src="js/modernizr.js"></script>
```

Comment:
this script refers to nothing and causing errors in your console: 

<img width="1109" alt="Screenshot 2022-11-11 at 4 07 26 PM" src="https://user-images.githubusercontent.com/40611703/201254903-c3cc9227-70e8-4a9f-8dc2-bb5d9ff07721.png">

You'll also see the error about your site.webmanifest. I've added a todo there, I'll look at that later